Code/GraphMol/Wrap/rough_test.py

```diff
@@ -8,19 +8,25 @@
 
 """
 
-import os, sys, tempfile, gzip, gc
-import unittest, doctest
-from rdkit import RDConfig, rdBase
-from rdkit import DataStructs
-from rdkit import Chem
-import rdkit.Chem.rdDepictor
-from rdkit.Chem import rdqueries
-
-from rdkit import __version__
-
+import doctest
+import gc
+import gzip
 # Boost functions are NOT found by doctest, this "fixes" them
 #  by adding the doctests to a fake module
 import importlib.util
+import os
```

Comment:
I understand why the automatic code formatter did this, but it really made the code less readable.
Please undo this change (so that the `import importlib.util` line and the comment before it are directly above line 30

Reply:
Oh! I'm sorry, I didn't even notice it did that. I just reverted it.